environment/image_recipes/docker/docker.json

```diff
@@ -89,6 +89,22 @@
         "-e bundle_flavour={{user `bundle_flavour`}}",
         "-e ansible_host=default"
       ]
+    },
+
+    {
+      "type": "file",
+      "source": "/rosdistro/rosdep/rosdep.yaml",
+      "destination": "/etc/ros/rosdep/rosdep.yaml"
+    },
+
+    {
+      "type": "shell",
+      "inline": [
+        "apt install -y python3-pip",
+        "python3 -m pip install setuptools -U",
```

Comment:
why are you installing setuptools here

Reply:
I've removed it. I rebuilt the image locally and it doesn't seem to be needed anymore.